src/error/kolibri_request.ts

```diff
@@ -0,0 +1,153 @@
+/*
+* Copyright 2021 HMS Industrial Networks AB
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http: //www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+export class KolibriRequestError extends Error {
```

Comment:
- Filename and Class name should be the same. Please rename this file to kolibri_request_error.ts
- This file should be exported in the root index.ts file so that the types are unable to the user.